.circleci/config.yml

```diff
@@ -1,48 +1,60 @@
-version: 2
+version: 2.1
+
+executors:
+  node:
+    parameters:
+      browsers:
```

Comment:
How do you think make this as string parameter `variant` which default value is "" and `-browsers` for specific case ?

Reply:
Not really usefull, there's really only `-browsers`: https://hub.docker.com/r/circleci/node

Reply:
https://circleci.com/docs/2.0/circleci-images/#language-image-variants

Reply:
I think it could make `<<# parameters.browsers >>-browsers<</ parameters.browsers >>` more readable with `<< parameters.variant >>`

Reply:
I dont agree. Introducing a text variant only makes it possible to introduce other errors because then you have to make certain to pass a proper string to that parameter.

If you want to support legacy browsers we can add a boolean legacy as well. If you want to support version tagging we can add a version param but with the same risk that it doesnt exists.

The idea behind defining an executor is that we can define it exactly to our needs so we dont have to duplicate settings anywhere else with the risk of them being wrong. Using a boolean for browsers provides that abstraction. With the current config I can guarantee you that however you call the executor the docker image exists. If we are going to pass a string you cannot guarantee that.

Reply:
I'm not quite understand your concern, anywhere which is using string parameters will have the risk you mentioned, so does it mean that the string parameters shouldn't exist at all ? As this config is for internal CI usage, I think it provide more flexibility and the risk you mentioned can be easily avoided/fixed by running a CI job.

I agree with your point about executor is for removing duplicate settings, but I think it does't mean the executor itself should be bigger and bigger because of more boolean params are added for ensuring stability, we still should care about the size and reusability of executor itself.

Maybe we can listen to the opinions of other members. @nuxt/core-team